src/contacts/processors/impl/AddNewOrganizationActionProcessor.java

```diff
@@ -0,0 +1,58 @@
+package contacts.processors.impl;
+
+
+import contacts.ConsoleReader;
+import contacts.Constants;
+import contacts.model.Organization;
+import contacts.model.Record;
+import contacts.processors.IRecordActionProcessor;
+import contacts.service.IRecordService;
+import contacts.service.Validator;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+public class AddNewOrganizationActionProcessor implements IRecordActionProcessor {
+
+    private final IRecordService personService;
+
+    public AddNewOrganizationActionProcessor(IRecordService personService) {
+
+        this.personService = personService;
+    }
+
+    @Override
+    public void doAction() throws IOException {
+        boolean isPhoneNumberValid = false;
+        String name = ConsoleReader.getStringFromConsole("Enter the organization name:");
+        String address = ConsoleReader.getStringFromConsole("Enter the address:");
+        String number = ConsoleReader.getStringFromConsole("Enter the phone number:");
+        isPhoneNumberValid = Validator.validatePhoneNumber(number);
+
+        Record record = new Organization(name, address);
+        record.setPerson(false);
```

Comment:
why do you need it?

Reply:
This is to separate the Record into a Person and an Organization?